plugin.rb

```diff
@@ -11,18 +11,15 @@
 
 after_initialize do
 
-  # black out spoilers in emails
-  Email::Styles.register_plugin_style do |fragment|
-    fragment.css(".spoiler").each do |spoiler|
-      spoiler["style"] = "color: #000; background-color: #000;"
+  on(:reduce_cooked) do |fragment, post|
+    fragment.css(".spoiler").each do |el|
+      link = fragment.document.create_element("a")
+      link["href"] = post.url
```

Comment:
isn't this supposed to be `post.full_url` ?

Reply:
It could be. It does not matter because there is [another step](https://github.com/nbianca/discourse/blob/1a7c47e4ce738701072dd0c36fb821622977d19b/lib/pretty_text.rb#L358) that ensures all URLs are absolute.